geoserve/geoserve.go

```diff
@@ -70,10 +70,21 @@ func NewServer(dbFile, license_key string) (server *GeoServer, err error) {
 	return
 }
 
-// Handle is used to handle requests from an HTTP server.  basePath is the path
+// HandleCors writes the access control allow origin for the lookup handler
+// from the ALLOW_ORIGIN environment variable. If empty, no access control is
+// not added to the response header.
+func HandleCors(resp *http.ResponseWriter, allowOrigin string) {
```

Comment:
Since this is used in only one place, I wouldn't put it in a separate function, and certainly not a publicly exported one.

Reply:
done. thanks!